File: 프로그래머스/0/181847. 0 떼기/0 떼기.cs
using System;
using System.Collections.Generic;

public class Solution 
{
    public string solution(string n_str) 
    {
        List<char> nstrList = new List<char>();
        int count = 0;
        for(int i=0 ; i<n_str.Length; i++)
        {
            if(n_str[i] != '0')
            {
                count++;
            }
            
            if(count >= 1)
            {
                nstrList.Add(n_str[i]);
            }
        }
        string result = new string(nstrList.ToArray());
        return result;
    }
}

File: 프로그래머스/0/181847. 0 떼기/0 떼기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string n_str) 
{
    while(true)
    {
        if(n_str[0] == '0')
        {
            n_str.erase(0,1);
        }
        
        if(n_str[0] != '0') { break; }
    }
    return n_str;
}